tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -187,6 +187,9 @@ spec:
         # files from an image, see https://github.com/enterprise-contract/ec-cli/issues/1109
         - name: EC_CACHE
           value: "false"
+      computeResources:
```

Comment:
I think we should try to come up with a way to make this configurable since it depends very much on how many images are being verified, and what kind of data is being loaded. Ideally, this is set at the pipeline level so the release service can adjust as needed.

(Not a blocker for this PR, of course)